mandatory/4-tax.js

```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(new_price) {
+  return concat ("£" + float(new_price * 1.20))
```

Comment:
There's some issues, here. Please request access to the solutions repo and check your answer. If there is something you don't understand, please let me know.